src/components/Properties/PropertyGroups.vue

```diff
@@ -39,15 +39,15 @@
 					:no-wrap="true"
 					:placeholder="t('contacts', 'Add contact in group')"
 					:multiple="true"
-					:taggable="true"
 					:close-on-select="false"
 					:disabled="isReadOnly"
-					:tag-width="60"
+					:clearable="true"
+					:deselectFromDropdown="true"
+					:taggable:="true"
+					:limit="3"
```

Comment:
Why limit to 3? This means that a user is not able to see all groups in the dropdown.

Reply:
Which comment? IMO, the limit should be removed.